spec/controllers/users/two_factor_authentication_controller_spec.rb

```diff
@@ -145,6 +146,19 @@ def index
         )
       end
 
+      it 'sends OTP via SMS for signing in' do
+        subject.current_user.encrypted_otp_secret_key = 'foo'
```

Comment:
This should be removed. As mentioned above, this is for TOTP users, and is making the test invalid.